src/app/rundown-view/services/action-trigger-producer-key-binding.service.ts

```diff
@@ -200,12 +205,17 @@ export class ActionTriggerProducerKeyBindingService implements KeyBindingService
     }
   }
 
-  private getBackgroundColorForSplitScreen(action: Tv2Action): string {
-    if (!this.isActionInsertIntoSplitScreenAction(action)) {
-      return SPLIT_SCREEN_COLOR
+  private getBackgroundColorForSplitScreen(actionTrigger: ActionTrigger<KeyboardTriggerData>, action: Tv2Action): string {
+    const overrideColor: string | undefined = actionTrigger.data.overrideColor
+    if (this.isActionInsertIntoSplitScreenAction(action)) {
+      return this.createSplitScreenGradient(overrideColor ?? this.getActionContentTypeBackgroundColor(action.metadata.insertedContentType), SPLIT_SCREEN_BOTTOM_COLOR)
     }
-    const topColor: string = this.getActionContentTypeBackgroundColor(action.metadata.insertedContentType)
-    return `linear-gradient(to bottom, ${topColor} 50%, var(--tv2-split-screen-lower) 50%)`
+    const topColor: string = overrideColor ?? SPLIT_SCREEN_TOP_COLOR
+    return this.createSplitScreenGradient(topColor, SPLIT_SCREEN_BOTTOM_COLOR)
```

Comment:
Minor thing: In the last scenario the topColor is extracted into a constant, but in the InsertIntoSplitScreenAction scenario, a more complex expression for topColor is not.

Reply:
I didn't want to deal with to variables called the same :p 

Sidenote: Updated from `overrideColor ?? SPLIT_SCREEN_TOP_COLOR` to `overrideColor || SPLIT_SCREEN_TOP_COLOR` since `overrideColor` is no longer optional.